### PR TITLE
wombex oracle deploy script added.

### PR DESCRIPTION
**Pull request by coolnft**
## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

**Review by vminkov — CHANGES_REQUESTED** [review]

File: packages/sdk/chainDeploy/mainnets/bsc.ts [hunk 1]
```diff
@@ -290,6 +290,17 @@ export const deploy = async ({ run, ethers, getNamedAccounts, deployments }: Cha
   const mpo = await ethers.getContract("MasterPriceOracle", deployer);
   const nativeBnb = underlying(assets, assetSymbols.BNB);
 
+  // Wombex Lp Token Price Oracle
```
Comment (carlomazzaferro) [inline packages/sdk/chainDeploy/mainnets/bsc.ts:293]: change name to wombat

File: packages/sdk/chainDeploy/mainnets/bsc.ts [hunk 2]
```diff
@@ -290,6 +290,19 @@ export const deploy = async ({ run, ethers, getNamedAccounts, deployments }: Cha
   const mpo = await ethers.getContract("MasterPriceOracle", deployer);
   const nativeBnb = underlying(assets, assetSymbols.BNB);
 
+  // Wombat Lp Token Price Oracle
+
+  const wombatOracle = await deployments.deploy("WombatLpTokenPriceOracle", {
+    from: deployer,
+    args: [mpo.address],
```
Comment (vminkov) [inline packages/sdk/chainDeploy/mainnets/bsc.ts:297]: please remove the `mpo.address` param and the ` waitConfirmations: 1,` config line

**Review by vminkov — CHANGES_REQUESTED** [review]

File: packages/sdk/chainDeploy/mainnets/bsc.ts [hunk 3]
```diff
@@ -290,6 +290,18 @@ export const deploy = async ({ run, ethers, getNamedAccounts, deployments }: Cha
   const mpo = await ethers.getContract("MasterPriceOracle", deployer);
   const nativeBnb = underlying(assets, assetSymbols.BNB);
 
+  // Wombat Lp Token Price Oracle
+
```
Comment (carlomazzaferro) [inline packages/sdk/chainDeploy/mainnets/bsc.ts:294]: nitpick: standarize spacing, & casing as in the other comment

**Review by carlomazzaferro — APPROVED** [review]

File: packages/chains/src/bsc/assets.ts [hunk 4]
```diff
@@ -477,6 +478,14 @@ const assets: SupportedAsset[] = [
     extraDocs: defaultDocs("https://bscscan.com", SD),
     disabled: true,
   },
+  {
+    symbol: assetSymbols["WOMBATLP-WBNB"],
+    underlying: WOMBATLP_WBNB,
+    name: "WombatLp WBNB",
+    decimals: 18,
+    oracle: OracleTypes.WombatLpTokenPriceOracle,
+    extraDocs: defaultDocs("https://bscscan.com", WOMBATLP_WBNB),
```
Comment (carlomazzaferro) [inline packages/chains/src/bsc/assets.ts:487]: Let's add custom docs that point to the BNB LP on Wombat

**Review by carlomazzaferro — APPROVED** [review]

**Pull request merged**